static/app/views/eventsV2/table/cellAction.tsx

```diff
@@ -430,21 +458,71 @@ class CellAction extends Component<Props, State> {
           </Reference>
           {menu}
         </Manager>
-      </MenuRoot>
+      </MenuActionButton>
+    );
+  }
+
+  renderContext() {
+    const {isContextPopperOpen: isOpen} = this.state;
+
+    if (!hasContext(this.props.dataRow, this.props.column)) {
+      return null;
+    }
+
+    const context = !isOpen
+      ? null
+      : createPortal(
+          <Popper placement="top" modifiers={this.modifiers}>
+            {({ref: popperRef, style, placement, arrowProps}) => (
+              <Menu
+                ref={ref => {
+                  (popperRef as Function)(ref);
+                  this.contextEl = ref;
+                }}
+                style={style}
+              >
+                <MenuArrow
+                  ref={arrowProps.ref}
+                  data-placement={placement}
+                  style={arrowProps.style}
+                />
+                <QuickContext dataRow={this.props.dataRow} column={this.props.column} />
+              </Menu>
+            )}
+          </Popper>,
+          this.portalEl
+        );
+
+    return (
+      <ContextActionButton>
+        <Manager>
+          <Reference>
+            {({ref}) => (
+              <MenuButton ref={ref} onClick={this.handleContextToggle}>
+                <IconShow size="sm" data-test-id="cell-action-context" color="subText" />
+              </MenuButton>
+            )}
+          </Reference>
+          {context}
+        </Manager>
+      </ContextActionButton>
     );
   }
 
   render() {
-    const {children} = this.props;
-    const {isHovering} = this.state;
+    const {children, organization} = this.props;
+    const {isHovering, isContextPopperOpen, isMenuPopperOpen} = this.state;
```

Comment:
Also, noticed when clicking on one of the menus to enable the popover, the other one disappears, I would expect the other one to still be visible, is that trivial to implement?